src/cryptography/hazmat/primitives/asymmetric/ec.py

```diff
@@ -252,6 +264,28 @@ def __init__(self, algorithm):
     algorithm = utils.read_only_property("_algorithm")
 
 
+@utils.register_interface(KeyDerivationFunction)
+class ECDHnoKDF(object):
```

Comment:
I think it is needed mostly for tests, I am not sure anyone should really use this w/o KDF, so I am ok leaving it out of the official API and just adding one in tests.